masterlist.yaml

```
@@ -3485,6 +3485,52 @@ plugins:
       - crc: 0x52109635
         util: 'SSEEdit v4.0.3h (Hotfix 1)'
 
+  - name: 'Praedy''s (SoulGems|SoulEssenceGem|ShrineOfAzuraSizeX).*\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/46372/'
+        name: 'MultiLayer Parallax Soulgems Azura''s star realm and holders HD'
+  - name: 'Praedy''s SoulGems.esp'
+    tag:
+      - Graphics
+      - Invent.Add
+      - Stats
+    clean:
+      - crc: 0x58BF838D
+        util: 'SSEEdit v4.0.3h (Hotfix 1)'
+  - name: 'Praedy''s SoulEssenceGem.esp'
+    tag:
+      - Graphics
+      - Stats
+    clean:
+      - crc: 0x661D16E8
+        util: 'SSEEdit v4.0.3h (Hotfix 1)'
+  - name: 'Praedy''s SoulGems ISC Patch.esp'
+    tag:
+      - Graphics
+      - Sound
+      - Stats
```

The graphics & stats changes are forwards from the main mod, so those tags aren't necessary.

```suggestion
    tag: [ Sound ]
```